### ISSUE TITLE
Cmake warns mr ! Huh ? (BB #769)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#769]
This issue was migrated from bitbucket.
**Original Reporter:** Xunil-Dev
**Original Date:** 03.01.2015 20:06:29 GMT+0000
**Original Priority:** minor
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/769

<hr>

Cmake give me a Warning ... 

```

```
# !c++

CMake Warning (dev) at src/game/CMakeLists.txt:57 (elseif):
  Policy CMP0054 is not set: Only interpret if() arguments as variables or
  keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.

  Quoted variables like "MSVC" will no longer be dereferenced when the policy
  is set to NEW.  Since the policy is not set the OLD behavior will be used.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at src/scripts/CMakeLists.txt:77 (elseif):
  Policy CMP0054 is not set: Only interpret if() arguments as variables or
  keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.

  Quoted variables like "MSVC" will no longer be dereferenced when the policy
  is set to NEW.  Since the policy is not set the OLD behavior will be used.
This warning is for project developers.  Use -Wno-dev to suppress it.

```

i use last version of cmake and Visual Studio 2013 Ultimate Edition (Licensed)
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Xunil-Dev
**Original Date:** 03.01.2015 20:07:02 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/769/#comment-14592203

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 11.01.2015 16:20:13 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/769/#comment-14737488

<hr>

Set cmake_policy for CMP0054 - Fix #769

→ <<cset 10cd438a3004>>
